package.json

```diff
@@ -17,11 +17,14 @@
     "argon2": "^0.28.3",
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
+    "http": "0.0.1-security",
```

Comment:
please `npm uninstall http`. node.js already has a built-in http module. idk wth this is.